### PR TITLE
Create goentri.com.stunning.json

### PR DESCRIPTION
**Pull request by evelyndearaujo-goentri**

### CODE REVIEW

File: goentri.com.stunning.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+   "variableDescription": "cnamePointsTo is the URL to where the subdomain is pointed and cnameHost is the host ",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%cnameHost%",
```
Comment (pawel-kow) [inline goentri.com.stunning.json:17]: why not `@` host and work with `host` parameter?
Yes, in the end you have the same record created, but some providers (IONOS, GoDaddy AFAIK) are identifying unique templates based on quadruple (domain, host, providerId, serviceId). In this case if you apply a second template with just different `%cnameHost%` the first one will be just removed.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: goentri.com.stunning.json (hunk 1 above)
Comment (evelyndearaujo-goentri) [reply goentri.com.stunning.json:17]: Yeah, makes sense. I would also need to change this template anyway, I'll make all those changes soon.

Comment (evelyndearaujo-goentri) [reply goentri.com.stunning.json:17]: Thanks

Comment (pawel-kow) [reply goentri.com.stunning.json:17]: So you want to keep this a1 group with CNAME here?

Comment (evelyndearaujo-goentri) [reply goentri.com.stunning.json:17]: qq - in case they send both a Record and CNAME would we be able to use 2 different templates for the same request?

Comment (pawel-kow) [reply goentri.com.stunning.json:17]: 2 templates in the same request are not possible in the protocol at the moment.
If you want to have both in the same request than this template is fine, with a downside of just one instance per domain. As you are setting A record on APEX I believe it's fine like this.

Comment (evelyndearaujo-goentri) [reply goentri.com.stunning.json:17]: Thanks @pawel-kow, I'm going to keep the template as it is.

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**